Dockerfile-slim.template

```diff
@@ -0,0 +1,66 @@
+FROM openjdk:8u181
```

Comment:
I think that we should try to make this more generic, like the Alpine image (e.g., `openjdk:8`).

As a side note, I tested this with a JRE based image vs. a JDK based one, and it fails to build (JRE image is usually a lot smaller, which is why we aim to use it when we can).

Reply:
Probably not. It may be doable, but there seem to be enough other fish to fry here.